cmd/manager/main.go

```diff
@@ -214,7 +215,12 @@ func newRootCommand() *cobra.Command {
 				log.Info("Starting the Cmd.")
 
 				// Start the Cmd
-				log.Fatal(mgr.Start(signals.SetupSignalHandler()))
+				err = mgr.Start(signals.SetupSignalHandler())
+				if err != nil {
+					log.Error(err)
+				}
+				// Canceling the leader election context
+				ch <- os.Interrupt
```

Comment:
We don't need a channel to do this any more. Just cancel the leader-election context directly.